samples/storage-mirror-3dc.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         requests:
           storage: 80Gi
   image:
-    name: cr.yandex/crptqonuodf51kdj7a7d/ydb:23.3.17
+    name: cr.yandex/crptqonuodf51kdj7a7d/ydb:23.2.12
```

Comment:
Check this please. Maybe we should use the last version from website "Downloads" page https://ydb.tech/docs/en/downloads/?